source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -152,6 +152,7 @@ class PoiSerializer(serpy.DictSerializer):
     administrative_regions = AdministrativeRegionsSerializer()
     poi_type = jsonschema.MethodField(display_none=False)
     properties = jsonschema.MethodField(display_none=False)
+    address = jsonschema.MethodField(display_none=False)
```

Comment:
we don't have a AddressSerializer?

Reply:
Yes we have but they read attributs directly in "properties.geocoding"